src/test/java/tests/RegistrationTest.java

```diff
@@ -52,7 +53,6 @@ public void registerNewUserTest() throws InterruptedException {
         signUpPage.inputRepeatedPassword(regData.getPassword());
         logger.info("Password has been repeated.");
         signUpPage.createAccountAccept();
-        Thread.sleep(10000);
-        Assert.assertEquals(logInPage.getSignInText(), "Sign In");
+        Assert.assertEquals(logInPage.getSignInText(), "to continue E-Restaurant");
```

Comment:
please provide meaningful error message

Reply:
Error messages are generating in Listener class at the moment. But test cases that are in this version are not correct, they was created for example to demo 3, and will be totally changed.